pkg/schedule/schedulers/balance_range.go

```diff
@@ -218,16 +580,35 @@ func NewRole(role string) Role {
 	}
 }
 
-func (r Role) String() string {
+func (r Role) getPeers(region *core.RegionInfo) []*metapb.Peer {
```

Comment:
I think it should be a method of region instead of role.

Reply:
done